src/index.html

```diff
@@ -1,13 +1,583 @@
+                       src="images/“.png" 
+                       alt="Air">
+                  </div>
+                </div>
+
+              </div>
+              <img class="pins" 
+                   src="images/“.png" 
+                   alt="pins">
+              <div class="testimonials-text">
+                AIR is an exceptional agency that leads with creative talent, first-class account servicing.
+              </div>
+              <div class="testimonials-name">Azadeh Hawkins</div>
+              <div class="department">
+                Hawkins Consulting
+              </div>
+            </div>
+            <div class="testimonials__area">
+              <div class="rec-box">
+                <div class="rec">
+                </div>
+                <div class="rec">
+                  <img class="rec-photo" 
+                       src="images/Rectangle__photo_3.png" 
+                       alt="Air">
+                  <div class="pins" 
+                       src="images/“.png" 
+                       alt="Air">
+                  </div>
+                </div>
+
+              </div>
+              <img class="pins" 
+                   src="images/“.png" 
+                   alt="pins">
+              <div class="testimonials-text">
+                AIR raises the agency bar to stratospheric heights on both creative output and client service.
+              </div>
+              <div class="testimonials-name">Michel Grover</div>
+              <div class="department">
+                Hulu
+              </div>
+            </div>
+
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section class="grid container shapes__block ">
+      <div class="section__grid">
+        <div class="section__block--title  vision ">
+          Vision, Passion, Results
+        </div>
+        <div class="section--title-discription  ">
+          We are sure that first-rate job is possible only if all three components are united.
+        </div>
+        <div class="button shape__button ">Apply</div>
+      </div>
+    </section>
+
+
+    <footer class=" container footer page__footer">
+      <div class="grid">
+
+        <div class="grid__item">
+          <div class=" page__footer--title" id="send_massage">
+            Send us a message
+          </div>
+          <form>
+            <input name="email" 
+                   type="email" 
+                   placeholder="Email" 
+                   class="contact-us__field form-field" 
+                   required>
+            <input name="name" 
+                   type="text" 
+                   placeholder="Name" 
+                   class="contact-us__field form-field" 
+                   pattern="\w+"
+              required>
+            <textarea name="message" 
+                      placeholder="Message" 
+                      class="form-field
+                             form-field--textarea">
+            </textarea>
```

Comment:
```suggestion
            <textarea 
              name="message" 
              placeholder="Message" 
              class="form-field form-field--textarea"
              ></textarea>
```
text started to enter not for edge of textarea
to fix it try to format a code as suggestion
also check styles for placeholder. not it not visible for textarea
![Screenshot from 2023-06-10 15-52-45](https://github.com/mate-academy/layout_dia/assets/33029864/0535886c-2e3b-4686-a810-2bec9a53668b)
![Screenshot from 2023-06-10 15-57-29](https://github.com/mate-academy/layout_dia/assets/33029864/d3bf62e1-70a9-4690-ab3d-7fe41451e306)